test/cli/run/empty-file.test.ts

```diff
@@ -0,0 +1,14 @@
+import { it, expect } from "bun:test";
+import { bunEnv, bunExe } from "harness";
+import { join } from "path";
+
+it("should execute empty scripts", () => {
+  const { stdout, exitCode } = Bun.spawnSync({
+    cmd: [bunExe(), "run", "--bun", join(import.meta.dir, "empty-file.js")],
+    env: bunEnv,
+    stderr: "inherit",
+  });
+
+  expect(stdout.toString().trim().endsWith("--pass--")).toBe(true);
```

Comment:
You should be checking if `stderr` is empty here instead. Use `"pipe"` instead of `"inherit"`.